### DISCUSSION TITLE
[example] todomvc subscriber not written

### DISCUSSION THREAD

**Discussion by chaudharisuresh997** [looplab/eventhorizon#358] [Q&A] [answered]
Can anyone please help me understand

-  how subsriber is written using pubsub.I assume it is missing in eventhorizon 
- How the projector consumes message from subscriber.
- did projector again save message in eventstore and apply events to aggregates.

**Comment by maxekman** [comment, marked as answer]
Not sure this is relevant any more. Closing for now.
